add timing fix

matthiasdiener opened the pull request:
See: https://github.com/pocl/pocl/pull/870

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

matthiasdiener committed:
add timing fix

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

matthiasdiener committed:
bump build number

matthiasdiener commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-build 3.20.2, conda-smithy 3.7.10, and conda-forge-pinning 2020.09.18.08.23.49

inducer commented:
LGTM, thanks!

matthiasdiener commented:
Something is wrong with the pocl-cuda package after this update:

```
$ conda install pocl-cuda
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /home/mdiener/miniforge3/envs/pocl

  added / updated specs:
    - pocl-cuda


The following NEW packages will be INSTALLED:

  ocl-icd            conda-forge/linux-64::ocl-icd-2.2.12-h516909a_1005
  pocl               conda-forge/linux-64::pocl-1.5-hb5afada_9        <-- incorrect
  pocl-cuda          conda-forge/linux-64::pocl-cuda-1.5-h926331d_9   <-- incorrect

```

`_9` is the wrong package version (should be `_10`: https://anaconda.org/conda-forge/pocl-cuda/files).

However, 

```
$ conda install pocl
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /home/mdiener/miniforge3/envs/pocl

  added / updated specs:
    - pocl


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    pocl-1.5                   |      hb5afada_10        17.5 MB  conda-forge
    ------------------------------------------------------------
                                           Total:        17.5 MB

The following NEW packages will be INSTALLED:

  ocl-icd            conda-forge/linux-64::ocl-icd-2.2.12-h516909a_1005
  pocl               conda-forge/linux-64::pocl-1.5-hb5afada_10  <-- correct

```

seems to install the right version.

The ppc64le version seems to have a similar issue.

@inducer @isuruf Do you know how to fix this?

inducer commented:
Build number 10 of `pocl-cuda` seems to exist: https://anaconda.org/conda-forge/pocl-cuda/files?sort=time&sort_order=desc

matthiasdiener commented:
> Build number 10 of `pocl-cuda` seems to exist: https://anaconda.org/conda-forge/pocl-cuda/files?sort=time&sort_order=desc

Right, which is why I'm confused why `conda install pocl-cuda` would install build number 9. On ppc64le, the same command installs build number 3...

matthiasdiener commented:
The plot thickens (?)

```
$ conda install pocl-cuda=1.5=h926331d_10
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /home/mdiener/miniforge3/envs/pocl

  added / updated specs:
    - pocl-cuda==1.5=h926331d_10


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    pocl-1.5                   |      hecece54_10        17.4 MB  conda-forge
    pocl-cuda-1.5              |      h926331d_10          79 KB  conda-forge
    ------------------------------------------------------------
                                           Total:        17.5 MB

The following NEW packages will be INSTALLED:

  ocl-icd            conda-forge/linux-64::ocl-icd-2.2.12-h516909a_1005
  pocl               conda-forge/linux-64::pocl-1.5-hecece54_10
  pocl-cuda          conda-forge/linux-64::pocl-cuda-1.5-h926331d_10

The following packages will be DOWNGRADED:

  libhwloc                                 2.1.0-h3c4fd83_0 --> 1.11.13-h3c4fd83_0

```

isuruf commented:
@matthiasdiener, can you open an issue in this git repo?
